### PR TITLE
fix location of cylc which is now cylc-flow

### PR DESCRIPTION
**Pull request by wxtim**
The change of project name from cylc to cylc-flow has broken the automated rose tests. This is a quick fix.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,9 @@ dist: xenial
 
 
 before_install:
-    - wget 'https://github.com/cylc/cylc/archive/master.tar.gz' -O '/tmp/cylc-master.tar.gz'
+    - wget 'https://github.com/cylc/cylc-flow/archive/master.tar.gz' -O '/tmp/cylc-master.tar.gz'
     - tar -xvf '/tmp/cylc-master.tar.gz' -C "${HOME}"
+    - mv "${HOME}/cylc-flow-master" "${HOME}/cylc-master"
```
Comment (wxtim) [inline .travis.yml]: This seems inelegant. Alternative suggestions gladly welcomed.

Comment (matthewrmshin) [reply .travis.yml]: OK for now. Maybe raise an issue later to address this (and any other instances where we reference *cylc*.

**Comment by matthewrmshin** [comment]
This is good. Please raise a similar fix for the 2019.01.x branch.

**Comment by matthewrmshin** [comment]
One review is good enough. Please raise a follow on issue to inspect the code base for any other potential references.

**Pull request merged**